src/OrchardCore.Modules/OrchardCore.Search/SearchRouteTransformer.cs

```diff
@@ -32,11 +32,13 @@ public override async ValueTask<RouteValueDictionary> TransformAsync(HttpContext
 
             if (searchProviders.Count() == 1)
             {
-                routeValue = new RouteValueDictionary(new {
+                routeValue = new RouteValueDictionary(new
+                {
                     Name = "Search",
                     Area = searchProviders.FirstOrDefault().AreaName,
                     Action = "Search",
-                    Controller = "Search" }
+                    Controller = "Search"
```

Comment:
`,` at the end

Reply:
Not necessary :)

Reply:
But why they made it available then ;)

Reply:
Seems hint for those who writing fast :)

Reply:
StyleCop has [this rule](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1413.md) to enforce a trailing comma in multi-line object initializers.

The advantage of having it is that changes to the code will be restricted to adding and removing lines, in contrast to adding a new line at the end of the object initializer AND adding a the comma to the line before. This makes diffs a lot more readable.

I think we should have analyzers in place for such things to avoid overhead in reviews.